### ISSUE TITLE
2.4.4 - CanPingPong is only running in editor instead of build and causing Excessive Pings disconnects.

### ISSUE THREAD

**Issue by DanielSnd** [FirstGearGames/FishNet#108]
In version 2.4.4, the file NetworkConnection.PingPong.cs method CanPingPong()

```csharp
        /// <summary>
        /// Called when a ping is received from this connection. Returns if can respond to ping.
        /// </summary>
        /// <returns>True to respond to ping, false to kick connection.</returns>
        internal bool CanPingPong()
        {
            /* Only check ping conditions in build. Editors are prone to pausing which can
             * improperly kick clients. */
#if !UNITY_EDITOR
            return true;
#else
```

The comment on the start of that method seems to indicate that it should only check for ping conditions in build, and not in editors as that leads to improperly kicking clients, but it seems to be doing the opposite, only doing it in editor and not in builds. I noticed it because I was getting kicked while testing in the editor due to "Excessive Pinging"

I changed it to 

```csharp
#if UNITY_EDITOR
            return true;
#else
```

**Comment by FirstGearGames** [comment]
Yep that's a bug. Thank you.

**Comment by FirstGearGames** [comment]
Will be resolved in 2.5.0
